boxsdk/client.py

```diff
@@ -124,17 +124,21 @@ def search(self, query, limit, offset, ancestor_folders=None, file_extensions=No
         :param ancestor_folders:
             Folder ids to limit the search to.
         :type ancestor_folders:
-            `list` of `unicode`
+            `list` of :class:`Folder`
         :param file_extensions:
             File extensions to limit the search to.
         :type file_extensions:
             `list` of `unicode`
+        :param mdfilters:
+            Filters used for metadata search
+        :type mdfilters:
+            :class:`MetadataSearchFilters`
         :return:
             A list of items that match the search query.
         :rtype:
             `list` of :class:`Item`
         """
-        return Search(self._session).search(query, limit, offset, ancestor_folders, file_extensions)
+        return Search(self._session).search(query, limit, offset, ancestor_folders, file_extensions, mdfilters)
```

Comment:
Actually, the above comment was in reference to the method definition (search.py has the same argument list).  For now I will leave as single line.